docs/cni/kubernetes/AWS.md

```diff
@@ -49,7 +49,7 @@ To check if you have a default VPC, run the following command, then save VPC ID
 aws ec2 describe-vpcs --filters "Name=isDefault,Values=true"
 
 # Save VpcId from output as environment variable (without quotes)
-VPC_ID=<VpcId>
+export VPC_ID=<VpcId>
```

Comment:
There's a bunch of instances below where you want an environment var saved off, but you've not put 'export' on the front.  e.g. 69, 75, 86, etc.